tests/masonry/builders/test_complete.py

```diff
@@ -387,22 +387,22 @@ def test_package_src():
 
 
 def test_package_with_include(mocker):
-    module_path = fixtures_dir / "with-include"
+    module_path = fixtures_dir / "with_include"
```

Comment:
Is this rename required? Personally, I like that these use `-` since they are not package directories.

Reply:
Mm, you're right. I also prefer `-` but the rest of the tests use `_` so I just wanted to maintain consistency.

Reply:
We can tackle this change in a separate PR if it is not required for the fix in this PR.